### PR TITLE
Move `import User` to runtime

### PR DESCRIPTION
**Pull request by ThibaudDauce**

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
Do you have some context on this change for future reference? :)

I wonder if it would lead to some performance downgrades for importing the User models at runtime for each request?

**Comment by ThibaudDauce** [comment]
> Do you have some context on this change for future reference? :)
> 
> I wonder if it would lead to some performance downgrades for importing the User models at runtime for each request?

In the dataservice PR I have some problems while importing the API force the import of `User` (you need the `api` module to create `fields` with `api.model()` and having `User` imported put a unwanted constraint. `import` are cache by Python, the performance downgrades should be negligeable I think.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - Move constants outside `models.py` files to `constants.py` [#3001](https://github.com/opendatateam/udata/pull/3001)
 - Move `db` and Mongo fields classes outside `udata.models` [#3005](https://github.com/opendatateam/udata/pull/3005)
 - Remove unused dependencies [#3006](https://github.com/opendatateam/udata/pull/3006)
+- Move some `import User` to runtime imports [#3013](https://github.com/opendatateam/udata/pull/3013)
```
Comment (MurdoqZodoq) [inline CHANGELOG.md:11]: Kuluwm/ All

**Review by MurdoqZodoq — APPROVED** [review]
(ANU)

**Review by maudetes — APPROVED** [review]
:+1:

**Pull request merged**